colcon_cargo/task/cargo/__init__.py

```diff
@@ -1,30 +1,36 @@
 # Copyright 2018 Easymov Robotics
 # Licensed under the Apache License, Version 2.0
 
+from typing import Optional
+
 import os
 import shutil
 
 from colcon_core.environment_variable import EnvironmentVariable
 
-"""Environment variable to override the Cargo executable"""
+# Environment variable to override the Cargo executable
```

Comment:
I'd keep this as docstring

Reply:
Depends what you use to generate the documentation, but if we're talking about sphinx, it doesn't support extracting docstrings from variables unfortunately